doc/source/conf.py

```diff
@@ -151,8 +151,15 @@ def __getattr__(cls, name):
     "versionwarning.extension",
     "sphinx_sitemap",
     "myst_parser",
+    "sphinx.ext.doctest",
+    "sphinx.ext.coverage",
 ]
 
+# There's a flaky autodoc import for "TensorFlowVariables" that fails depending on the doc structure / order
+# or imports.
+autodoc_mock_imports = ["ray.experimental"]
```

Comment:
hmm will this make it so that all 'ray.experimental' objects don't get rendered properly?

Reply:
yes, this was just one occurrence though. I've removed it from docs, as it didn't add any value. This way we don't need the above hack.